packages/app/.eslintrc.json

```diff
@@ -2,5 +2,8 @@
     "extends": "../../config/.eslintrc.json",
     "parserOptions": {
         "project": "tsconfig.json"
+    },
+    "rules": {
+        "import/no-default-export": "off"
```

Comment:
Unless it's used everywhere (I didn't check), I'd recommend adding a `// eslint:disable-next-line` (this is TSLint format, but you know what I mean) comment to the default export lines instead of disabling the rule for the whole package.

Reply:
That's true. Will update before merge.